### DISCUSSION TITLE
[Feature Request]: How edit robots.txt?

### DISCUSSION THREAD

**Discussion by Grosso2020** [GetPublii/Publii#1366] [Help] [answered]
### Feature Description

Hello.
Thank you for an excellent CMS for your blog. Very convenient for me. My question is how to edit robots.txt so that the CMS does not overwrite the file?

**Comment by bobmitro** [comment, marked as answer]
To modify a website's robots.txt file in Publii, create a custom robots.txt file directly in the root directory using a File Manager tool.

**Reply by raramuridesign** [reply]
@bobmitro There are new settings which add exclusion to openai. If this is selected does this overwrite the file? 
If yes, would we then need to add the changes to robots.txt again ?
Just require a clarification
M.

**Reply by bobmitro** [reply]
The custom robots.txt file overrides all settings, so you have to add any needed rules to this custom file.

**Reply by raramuridesign** [reply]
@bobmitro Thanks. I think it is important that this is highlighted, so users know that a custom file will not be affected in anyway.
